manifests/mod/passenger.pp

```diff
@@ -750,14 +742,6 @@
         fail("Passenger config option :: passenger_thread_count is not introduced until version 4.0.0 :: ${passenger_installed_version} is the version reported")
       }
     }
-    if $passenger_use_global_queue {
-      if (versioncmp($passenger_installed_version, '4.0.0') > 0) {
-        fail('REMOVED PASSENGER OPTION :: passenger_use_global_queue :: -- no message on the current passenger reference webpage -- ')
-      }
-      if (versioncmp($passenger_installed_version, '2.0.4') < 0) {
-        fail("Passenger config option :: passenger_use_global_queue is not introduced until version 2.0.4 :: ${passenger_installed_version} is the version reported")
-      }
-    }
     if $passenger_user {
       if (versioncmp($passenger_installed_version, '4.0.0') < 0) {
```

Comment:
If you're raising the minimum level to 4.x you can drop this.

Reply:
yeah, you are right, we should be cleaning up all < 5.0.0 based on the my approach as of now, coz we are expecting these are params coming from some older versions.